shell/gpu/gpu_surface_gl.h

```diff
@@ -21,6 +22,14 @@ class GPUSurfaceGLDelegate {
   virtual bool GLContextPresent() = 0;
 
   virtual intptr_t GLContextFBO() const = 0;
+
+  virtual sk_sp<GrContext> GetGrContext() const {
```

Comment:
Yes, this allows me to keep this CL Android only defer changing the other backends to later CLs.